src/Core/Settings.h

```diff
@@ -339,7 +339,7 @@ struct Settings : public SettingsCollection<Settings>
     \
     M(SettingUInt64, max_memory_usage, 0, "Maximum memory usage for processing of single query. Zero means unlimited.", 0) \
     M(SettingUInt64, max_memory_usage_for_user, 0, "Maximum memory usage for processing all concurrently running queries for the user. Zero means unlimited.", 0) \
-    M(SettingUInt64, max_memory_usage_for_all_queries, 0, "Maximum memory usage for processing all concurrently running queries on the server. Zero means unlimited.", 0) \
+    M(SettingUInt64, max_server_memory_usage, 0, "Maximum memory usage for server. It can be specified only for default profile.", 0) \
```

Comment:
Maybe `Only has meaning at server startup.` will be better (like other settings has)?